scripts/os_configure/setup_auto_sourcing.bash

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# Copyright 2021 Denis Stogl (Stogl Robotics Consulting)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+script_own_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
+source $script_own_dir/../../setup.bash
+
+user_decision "Do you want to automatically source RosTeamWs? This is going to add a .ros_team_ws file to your home directory. Additionally your .bashrc is modified to automatically source RosTeamWs."
+if [[ " ${negative_answers[*]} " =~ " ${user_answer} " ]]; then
+    print_and_exit "Aborting the setup of autosourcing. Exiting..."
+fi
+
+echo "Copying .ros_team_ws_rc to your home folder."
+
+rtw_file=".ros_team_ws_rc"
+rtw_file_location=~/"$rtw_file"
+template_location="$FRAMEWORK_BASE_PATH/templates/$rtw_file"
+if ! [ -f "$HOME/$rtw_file" ]; then
+    cp "${template_location}" ~/.
+fi
+sed -i "s|source <PATH TO ros_team_workspace>/setup.bash|source $FRAMEWORK_BASE_PATH/setup.bash|g" "$rtw_file_location"
+
+echo ""
+user_decision "Do you want to add add colored output to your shell? This will show the current git branche, ros workspace and time in terminal. If you are unsure you can test this later by uncommenting the \"export PS1=\" in the .ros_team_ws_rc file or rerun the setup-auto-sourcing command."
```

Comment:
```suggestion
user_decision "Do you want to add add colored output to your shell? This will show the current git branch, ros workspace and time in terminal. If you are unsure you can test this later by uncommenting the \"export PS1=\" in the .ros_team_ws_rc file or rerun the setup-auto-sourcing command."
```

Reply:
Done